docs/source/api.rst

```diff
@@ -289,6 +593,27 @@ Photovoltaic
 .. literalinclude:: schemas/request/Photovoltaic.json5
    :language: javascript
 
+Each solar electric photovoltaic (PV) system is entered in ``...building.systems.photovoltaic``. If not entered, the simulation will not include photovoltaics.
+
+=================================  =======  ==================  ==============  ========  ==================  ============================================== 
+Property                           Type     Units               Constraints     Required  Default             Notes
+=================================  =======  ==================  ==============  ========  ==================  ==============================================       
+``id``                             id                           Must be unique  yes       not defined 
+``location``                       string                       see [#]_        no        "roof"  
+``moduleType``                     string                       see [#]_        no        "standard"  
+``tracking``                       string                       see [#]_        no        "fixed" 
+``arrayOrientation``               string   direction           see [#]_        yes     
+``arrayTilt``                      float    degrees             0-90            yes                           Tilt relative to horizontal
+``maxPowerOutput``                 float    W                   >0              yes     
+``inverterEfficiency``             float    fraction            0-1             no        0.96    
+``yearModulesManufactured``        integer  year                >1600           no                            Informs systemLossesFraction, which defaults to 0.14
+=================================  =======  ==================  ==============  ========  ==================  ============================================== 
+
+.. [#] ``location`` choices are "ground" and "roof".
+.. [#] ``moduleType`` choices are "standard", "premium", and "thin film".
+.. [#] ``tracking`` choices are "fixed", "1-axis", "1-axis backtracked", and "2-axis".
+.. [#] ``arrayOrientation`` choices are "northeast", "east", "southeast", "south", "southwest", "west", "northwest", and "north".
```

Comment:
Because of the problem above, footnotes got shifted several positions.

![image](https://github.com/RadiantLabs/modeling-api-docs/assets/10285345/affbe08d-ce1a-4b2b-9bf4-277eb924e257)

I'll look for the docs and see if I can solve this issue.